pkg/inventory/inventory.go

```diff
@@ -0,0 +1,173 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package inventory
+
+import (
+	"encoding/json"
+
+	"sigs.k8s.io/kustomize/pkg/resid"
+)
+
+// A Refs is a map from an Item string to a list of Items
+// that it is referred
+type Refs map[string][]resid.ItemId
```

Comment:
`map[ItemId][]ItemId`

Reply:
I used `encoding/json` to marshal the Inventory object into bytes. `map[ItemId][]ItemId` is not supported.